proxy-node-translator/src/main/java/uk/gov/ida/notification/translator/saml/HubResponseTranslator.java

```diff
@@ -135,18 +135,19 @@ private static String getCombinedSurnames(HubResponseContainer hubResponseContai
     }
 
     private static String getLatestValidDateOfBirth(HubResponseContainer hubResponseContainer) {
-        var datesOfBirth = hubResponseContainer.getAttributes().getDatesOfBirth();
+        var datesOfBirth = hubResponseContainer.getAttributes().getDatesOfBirthAttributesList();
         var validDatesOfBirth = datesOfBirth.getValidAttributes();
         if (validDatesOfBirth.isEmpty()) {
             throw new HubResponseTranslationException("No verified current date of birth present: " + datesOfBirth.createAttributesMessage());
         }
+
         return validDatesOfBirth.stream()
-                .map(Attribute::getValue)
-                .reduce(BinaryOperator.maxBy(DateTimeComparator.getDateOnlyInstance()))
-                .map(Attributes::getDateInEidasFormat).get();
+                .map(NonMatchingVerifiableAttribute::getValue)
+                .reduce(BinaryOperator.maxBy(Comparator.comparing(LocalDate::toEpochDay)))
+                .map(date -> date.toString()).get();
```

Comment:
Why was the `getDateInEidasFormat` logic removed? There's a specific date format for eIDAS that we need to comply with.

Reply:
Previously we were using a joda date object, now it's a java LocalDate. Calling `toString` on LocalDate gives us the correct format, which is nice!

Reply:
OK fine 👍